presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -1363,6 +1363,20 @@ public static long widthBucket(@SqlType(StandardTypes.DOUBLE) double operand, @S
         return lower;
     }
 
+    @Description("The bucket name of a value given an array of bins and corresponding bin names")
+    @ScalarFunction("width_bucket")
```

Comment:
@amitkdutta This function doesn't appear to be a math function, but it is in MathFunctions.java. Why does it do? Can it be replaced by a composition of some existing functions?

Reply:
Its an extension of already existing width_bucket function. Since width_bucket is a math function, I added the extension here as well.